cmd/uplink/cmd/put.go

```diff
@@ -28,7 +28,7 @@ func putMain(cmd *cobra.Command, args []string) (err error) {
 
 	ctx := process.Ctx(cmd)
 
-	u0, err := utils.ParseURL(args[0])
+	u, err := fpath.New(args[0])
```

Comment:
Some better name here like `src`, `input` or similar.

Reply:
Changed it everywhere in all commands.